real_time/arduino_src/lib_avr/uart/uart.h

```diff
@@ -121,7 +121,7 @@ Date        Description
  * Enable UARTs per project in the makefile with gcc -D flags.
  * UART0 is always enabled because library does not compile correctly without.
  */
-#define USART0_ENABLED
+// #define USART0_ENABLED
```

Comment:
It through a warning on compilation (double definition), since we define `USART0_ENABLED` using the Makefile. Apparently it was necessary for the library to compile, I don't know why.